grails-app/controllers/org/pih/warehouse/inventory/InventoryController.groovy

```diff
@@ -1402,9 +1402,9 @@ class InventoryController {
             def row = [
                     'Product code'    : it.product?.productCode,
                     'Product name'    : it.product?.name,
-                    'Lot number'      : StringEscapeUtils.escapeCsv(it.inventoryItem?.lotNumber ?: ""),
+                    'Lot number'      : it.inventoryItem?.lotNumber,
```

Comment:
So now the row would look something like this:
```
def row = [
    productCode: [label: g.message(code: "product.productCode.label"), value: it?.product?.productCode],
    productName: [label: g.message(code: "product.name.label"), value: it?.product?.productCode],
    expirationDate: [label: g.message(code: "inventoryItem.expirationDate.label"), value: it.inventoryItem?.expirationDate?.format("MM/dd/yyyy"), format: "MM/dd/yyyy"]
]
```
You could also pull the label and format out into a separate data structure so we're not duplicating that data on each row.
```
def columns = [
    productCode: [label: g.message(code:"product.productCode.label"), format: "MM/dd/yyyy"]
]
```